web3_proxy/src/stats/influxdb_queries.rs

```diff
@@ -575,7 +576,25 @@ pub async fn query_user_stats<'a>(
                             );
                         }
                         _ => {
-                            error!("error_response should always be a Long!");
+                            error!("error_response should always be a String!");
+                        }
+                    }
+                } else if stat_response_type == StatType::Detailed && key == "user_error_response" {
```

Comment:
just confirming that this was added to the influx schema

Reply:
Don't we use a schemaless influx that gets set on first insert?

Reply:
i suppose we do